instantswap/exchange/trocador/trocador.go

```diff
@@ -0,0 +1,254 @@
+package trocador
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/crypto-power/instantswap/instantswap"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+const (
+	API_BASE = "https://trocador.app/api/"
+	LIBNAME  = "swapzone"
+)
+
+type trocador struct {
+	client *instantswap.Client
+	conf   *instantswap.ExchangeConfig
+}
+
+func init() {
+	/*instantswap.RegisterExchange(LIBNAME, func(config instantswap.ExchangeConfig) (instantswap.IDExchange, error) {
+		return New(config)
+	})*/
+}
+
+// SetDebug set enable/disable http request/response dump.
+func (t *trocador) SetDebug(enable bool) {
+	t.conf.Debug = enable
+}
+
+// New return a SwapZone client.
```

Comment:
change all to instances to trocador
```suggestion
// New return a trocador client.
```